packages/integration-tests/tests/bridge_erc20.ts

```diff
@@ -566,6 +571,22 @@ describe('Bridging ERC20 tokens', async function () {
       expect(currentWithdrawnAmountAfterSettingLimit === 0n).to.be.true;
     });
 
+    it('Check the remaining token balance on Fuel', async () => {
+      // fetch the remaining token balance
+      const currentTokenBalance = await fuelTokenSender.getBalance(
+        fuel_testAssetId
+      );
+
+      // currentTokenBalance has BN type by default hence the use of BN for conversion here
+      const expectedRemainingTokenBalanceOnFuel =
+        tokenBalanceBeforeWithdrawingOnFuel.sub(
+          new BN(((NUM_TOKENS * 2n) / DECIMAL_DIFF).toString())
```

Comment:
Can we move this up so that it does not become dependant on how many withdrawals have happened before?

Reply:
shouldn't we check the remaining token balance after the partial withdrawals have happened ?